.env.test

```
@@ -0,0 +1,17 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+
+# Details for Trello API Credentials:
+TRELLO_API_KEY=TESTAPIKEY
+TRELLO_API_TOKEN=TESTAPITOKEN
+TRELLO_BOARD_ID=56851f21489d3cebc979ded
+TRELLO_TODO_LIST_ID=656851f20f4a7c4638899308
+TRELLO_ACTIVE_LIST_ID=656851f238936766c85715d9
+TRELLO_DONE_LIST_ID=656851f2348b0a7bcf295353
```

Even though these aren't sensitive, I'm still a fan of using blatantly fake variables - if nothing else it will help make it more obvious if your tests accidentally start using real values:
```suggestion
TRELLO_BOARD_ID=board-id
TRELLO_TODO_LIST_ID=todo-list-id
TRELLO_ACTIVE_LIST_ID=active-list-id
TRELLO_DONE_LIST_ID=done-list-id
```